### DISCUSSION TITLE
Map - Popup item doesn't show

### DISCUSSION THREAD

**Discussion by nunoafdinis** [naverz/zepeto-studio-kor#650] [5. Q&A]
Hi,

We are a textile company and we are trying to create a custom world to show and "sell" our products.
We are using Unity with ZEPETO tools and following your tutorials to create them. We already have a button that when clicked shows a popup with our custom item. However, when we open the world in Zepeto app (through QR Code in Unity), none popup is showed when we click on button.

Can you please help?

Thank you,
Nuno Dinis
________________________________________

https://user-images.githubusercontent.com/5837601/164426383-6015c251-535f-4c66-a3f5-2a27a74697b2.mov

https://user-images.githubusercontent.com/5837601/164426394-041757b8-654b-44a8-ae1a-1bbe6d0cc3f3.mp4

**Comment by Zepeto-TechEvangelist** [comment]
Hello, there's a pop-up window on the Unity editor, but it doesn't appear on mobile, right? 
(I can't see your Unity editor video)
It would be helpful if you attach a code/screen shot.
and ZEPETO World Package Version.

**Reply by nunoafdinis** [reply]
Hi,

First of all, thank you for you help.

I have attached a video and an image (using IWP Button).
Please let me know if you need some more elements to try to understand our problem.


https://user-images.githubusercontent.com/5837601/164679858-c1bbd26b-41c0-4c2c-ae12-4fb0112fb5db.mp4
<img width="960" alt="unity" src="https://user-images.githubusercontent.com/5837601/164680483-1031440b-ccb0-4a47-b2a3-d288fe800dbb.PNG">

**Reply by Zepeto-TechEvangelist** [reply]
Now I can watch your video.
We will prepare to solve the problem.
Thank you.

**Reply by nunoafdinis** [reply]
Hi,

Thank you for your support.
Please let me know when you have any solution for us.

**Reply by nunoafdinis** [reply]
Hi,

We are pushing harder to try solve this issue.
We have tried this sample project (https://github.com/naverz/zepeto-script-sample) and the behavior remains the same: any popup shows in Unity (item pop, error message pop, etc) but in mobile (iPad, Samsung tab) doesn't.

So, this behaviour happens in all popups and not only on item popup.
